pkg/proxy/service.go

```diff
@@ -324,6 +324,7 @@ func (sct *ServiceChangeTracker) Update(previous, current *v1.Service) bool {
 	// if change.previous equal to change.current, it means no change
 	if reflect.DeepEqual(change.previous, change.current) {
 		delete(sct.items, namespacedName)
+		klog.V(5).InfoS("Service update does not affect proxying", "service", klog.KObj(svc))
```

Comment:
same comment as before, maybe having everything at level 4 can be more consistent